scripts/reboot

```diff
@@ -260,4 +276,18 @@ if [ -x ${DEVPATH}/${PLATFORM}/${PLAT_REBOOT} ]; then
 fi
 
 VERBOSE=yes debug "Issuing OS-level reboot ..." >&2
-exec /sbin/reboot $@
+echo "Going to issue reboot command..."
```

Comment:
echo not necessary here, there is a verbose debug above

Reply:
Removed it in revised PR change set.